Vizzuality/heco-invest
# Add update user information form

This PR adds the update user information form

OBS: the design have an avatar picture and an image uploader button, but the user form doesn’t have the picture and it is not in the scope of the task. For that reason I let that section commented for future implementation.

## Testing instructions

Go to 'my preferences'/information area.

- You should be able to update your name and last name.
- Both fields are mandatory and display an error if empty 
- The email is visible but is disabled for edition
- If you click on the 'Update password' button, a modal is displayed to confirm the action.

## Tracking
[LET-1066]https://vizzuality.atlassian.net/browse/LET-1066


[LET-1066]: https://vizzuality.atlassian.net/browse/LET-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
